src/components/Footer.jsx

```diff
@@ -41,34 +45,52 @@ class SiteFooter extends React.Component {
       intl,
     } = this.props;
     const showLanguageSelector = supportedLanguages.length > 0 && onLanguageSelected;
-    const { config } = this.context;
+    const config = getConfig();
 
     return (
-      <footer
-        role="contentinfo"
-        className="footer d-flex border-top py-3 px-4"
-      >
-        <div className="container-fluid d-flex">
-          <a
-            className="d-block"
-            href={config.LMS_BASE_URL}
-            aria-label={intl.formatMessage(messages['footer.logo.ariaLabel'])}
-          >
-            <img
-              style={{ maxHeight: 45 }}
-              src={logo || config.LOGO_TRADEMARK_URL}
-              alt={intl.formatMessage(messages['footer.logo.altText'])}
-            />
-          </a>
-          <div className="flex-grow-1" />
-          {showLanguageSelector && (
-            <LanguageSelector
-              options={supportedLanguages}
-              onSubmit={onLanguageSelected}
-            />
-          )}
-        </div>
-      </footer>
+      <div className="wrapper wrapper-footer">
+        <footer id="footer" className="tutor-container">
+          <div className="footer-top">
+            <div className="powered-area">
+              <ul className="logo-list">
+                <PluginSlot id="poweredby_text_slot"><li>{intl.formatMessage(messages['footer.poweredby.text'])}</li></PluginSlot>
+                <PluginSlot id="tutor_logo_slot">
+                  <li>
+                    <a href="https://docs.tutor.overhang.io" rel="noreferrer" target="_blank">
+                      <Image
+                        src={`${config.LMS_BASE_URL}/theming/asset/images/tutor-logo.png`}
+                        alt={intl.formatMessage(messages['footer.tutorlogo.altText'])}
+                        width="57"
+                      />
+                    </a>
+                  </li>
+                </PluginSlot>
+                <PluginSlot id="openedx_logo_slot">
+                  <li>
+                    <a href="https://open.edx.org" rel="noreferrer" target="_blank">
+                      <Image
+                        src={logo || `${config.LMS_BASE_URL}/theming/asset/images/openedx-logo.png`}
+                        alt={intl.formatMessage(messages['footer.logo.altText'])}
+                        width="79"
+                      />
+                    </a>
+                  </li>
+                </PluginSlot>
+              </ul>
```

Comment:
I think this format is a little too prescriptive. We shouldn't be including a `tutor` logo/slot by default, because not everyone uses Tutor to deploy it. And this kind of assumes there will be either two logos, one logo, or none. What would be much more flexible is something like this:

```ts
        <PluginSlot as="footer" id="mfe-global-footer" className="footer border-top py-3 px-4" role="contentinfo">
            <PluginWidget id="default-poweredby-logo"><a ...><img ... /></a></PluginWidget>
            <PluginWidget id="language-selector"><a ...><img ... /></a></PluginWidget>
            <PluginWidget id="copyright-text"><a ...><img ... /></a></PluginWidget>
        </PluginSlot>
```

And then for your Tutor plugin, in the config you have something like:
1. `Hide` the `default-poweredby-logo` widget.
2. Insert the new `poweredby-tutor-openedx` widget `before` the `default-poweredby-logo` widget.

But @jsnwesson is it possible to define several default widgets in one slot like this with the plugin framework?

Reply:
Plugin Framework currently does not have any component named `PluginWidget`. In the latest commit, I've added `PluginSlot` instead of `PluginWidget`. Can you please check again? 
Furthermore, on what basis, we are going to decide whether we have to use PluginSlot or PluginWidget (if it will be added in plugin framework)? 
What I understand is that a list is PluginSlot for example, and its items will be Widgets. Here, in footer, all are different elements (logo, lang selector, copyright) and not list. Then, why are we treating them like widgets? Any point which I'm missing.

Reply:
Hi @hinakhadim and @bradenmacdonald , huge apologies for the delay in response! 
I think one of the confusions here was the naming convention differences between Braden's UI Plugin approach and Frontend Plugin Framework's ("FPF"). 

- In FPF, there are only `PluginSlots`, not `PluginWidgets` (and there is no plan to bring that name into the library for any use case). 
- FPF takes Braden's UI approach of using PluginOperations to define what widgets will go into the PluginSlot as defined by the `env.config.js` file.
- Lastly, and possibly most importantly, any content inside a `PluginSlot` is considered the `defaultContent`. That `defaultContent` is treated as one `children` body and can either be included or hidden in production by modifying the `keepDefault` attribute for that PluginSlot via `env.config.js`. ([Source](https://github.com/openedx/frontend-plugin-framework?tab=readme-ov-file#default-content))

Reply:
And to confirm with you @hinakhadim , I noticed that inside the "parent" PluginSlot with the id `mfe-global-footer`, I see that it has four PluginSlots as part of its `defaultContent` body. 
Is the desire here so that each of those four "child" PluginSlots can be modified in the `env.config.js`, or was it there a misunderstanding around if they each needed to be wrapped by a PluginSlot? 
For example, if you didn't want to customize each individual component in the `defaultContent` of the "parent" PluginSlot, you could rewrite this as: 
```
<PluginSlot as="footer" id="mfe-global-footer" className="footer border-top py-3 px-4" role="contentinfo">
  <a
    className="d-block"
    href={config.LMS_BASE_URL}
    aria-label={intl.formatMessage(messages['footer.logo.ariaLabel'])}
  >
    <img
      style={{ maxHeight: 45 }}
      src={logo || config.LOGO_TRADEMARK_URL}
      alt={intl.formatMessage(messages['footer.logo.altText'])}
    />
  </a>
  <FooterLinks />
  {showLanguageSelector && (
    <LanguageSelector
      options={supportedLanguages}
      onSubmit={onLanguageSelected}
    />
  )}
  <span className="copyright-site mt-2">{intl.formatMessage(messages['footer.copyright.text'])}</span>
</PluginSlot>
```

Reply:
@hinakhadim @jsnwesson  Sorry for the confusion. I was hoping there would be some way to have multiple widgets in a plugin slot in FPF. It seems like is it possible if you define it in a config, but it's not currently possible to define multiple widgets as the default content without setting a config override.

I do feel that for a use case like this, it's nice to have several default widgets in one slot, but you could either just create them as several slots inside a bigger slot or one slot with one default content that is completely overwritten by a plugin even though that may cause a lot of duplicate code.

Reply:
> Is the desire here so that each of those four "child" PluginSlots can be modified in the env.config.js, or was it there a misunderstanding around if they each needed to be wrapped by a PluginSlot?

@jsnwesson My thinking was that when someone wants to change only logo, they can target the logo PluginSlot. if someone doesn't require this footer and wants to introduce their own they can target the parent PluginSlot.

As @bradenmacdonald said  previously, giving option for only one bigger slot causes lot of duplications. Someone wants to change logo has to completely override the whole footer component which causes frustration. What are your thoughts, Jason?

Reply:
Knowing that this PluginSlots within a PluginSlot implementation is intentional, I think it works! My team has a similar thing in mind for the Course List module in Learner Dashboard MFE.